practice-one/src/styles/bases/base.css

```diff
@@ -0,0 +1,35 @@
+/* Import font */
```

Comment:
Do you know `reset css`?

Reply:
yes, i know. but, i think i wrong here. It isn't reset css